File: app/src/main/java/com/eftimoff/bakingapp/recipelist/view/RecipeListActivity.java
package com.eftimoff.bakingapp.recipelist.view;

import android.os.Bundle;
import android.support.annotation.Nullable;

import com.eftimoff.bakingapp.R;
import com.eftimoff.bakingapp.app.view.BaseActivity;

public class RecipeListActivity extends BaseActivity {

    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        if (savedInstanceState == null) {
            getSupportFragmentManager().beginTransaction()
                    .replace(R.id.fragment_container, RecipeListFragment.newInstance())
                    .commit();
        }
    }

    @Override
    protected int getLayoutRes() {
        return R.layout.activity_recipe_list;
    }

}

File: app/src/main/java/com/eftimoff/bakingapp/recipelist/view/RecipeListActivity.kt
package com.eftimoff.bakingapp.recipelist.view

import android.os.Bundle
import com.eftimoff.bakingapp.R
import com.eftimoff.bakingapp.app.extensions.replace
import com.eftimoff.bakingapp.app.view.BaseActivity

class RecipeListActivity : BaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        if (savedInstanceState == null) {
            supportFragmentManager.replace(RecipeListFragment.newInstance())
        }
    }

    override fun getLayoutRes(): Int {
        return R.layout.activity_recipe_list
    }
}
